sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -598,7 +598,7 @@ logger.setLevel(logging.DEBUG)
 handler = logging.StreamHandler(stream=sys.stdout)
 logger.addHandler(handler)
 
-endpoint = "https://<region>.cognitiveservices.azure.com/"
+endpoint = "https://<name>.cognitiveservices.azure.com/"
```

Comment:
curious about this change. I am looking at the readme and I see places where `region` or `my-custom-subdomain` is used. Should all this be the same?

Reply:
We've always shown both in the README and have a note that says

> Note that if you create a custom subdomain name for your resource the endpoint may look different than in the above code snippet. For example, https://{my-custom-subdomain}.cognitiveservices.azure.com/.

The reason I changed it here to "name" is because we use DefaultAzureCredential() in the sample which must be used with a custom subdomain name. But I think I will change it to <my-custom-subdomain> to be consistent, thanks!